nibabel/testing/__init__.py

```diff
@@ -62,6 +65,22 @@ def assert_allclose_safely(a, b, match_nans=True, rtol=1e-5, atol=1e-8):
     assert_true(np.allclose(a, b, rtol=rtol, atol=atol))
 
 
+def check_iteration(iterable):
```

Comment:
Then maybe discard this function and use `[t for t in tractogram]` instead of `assert_true(check_iteration(tractogram))` ?  I don't feel strongly, but the boolean for whether an error was raised seems a bit fishy, and it seems better to let the error arise in the test function rather than be eaten up here in the helper function.